src/network/networktool.cpp

```diff
@@ -12,39 +12,39 @@
 
 PING_LOGGING_CATEGORY(NETWORKTOOL, "ping.networktool")
 
-
-QString NetworkTool::_gitUserRepo = "bluerobotics/ping-viewer";
-QString NetworkTool::_gitUserRepoFirmware = "bluerobotics/ping-firmware";
-
+namespace {
+    QString gitUserRepo = QStringLiteral("bluerobotics/ping-viewer");
+    QString gitUserRepoFirmware = QStringLiteral("bluerobotics/ping-firmware");
+    const QString gitReleases = QStringLiteral("https://api.github.com/repos/%1/releases");
+    const QString repoContents = QStringLiteral("https://api.github.com/repos/%1/contents/%2");
+    const QString projectTag = QStringLiteral(GIT_TAG);
+    const QString userRepoUrl = QStringLiteral(GIT_URL);
+}
 NetworkTool::NetworkTool()
 {
     //*github.com/user/repo* results in user/repo
     const static QRegularExpression regex(R"(github.com\/([^.]*))");
-    QRegularExpressionMatch regexMatch = regex.match(QStringLiteral(GIT_URL));
+    QRegularExpressionMatch regexMatch = regex.match(userRepoUrl);
     if (!regexMatch.hasMatch() || regexMatch.lastCapturedIndex() < 1) {
-        qCWarning(NETWORKTOOL) <<
-                               "Fail to get github user and repository! "
-                               "It will not be possible to check for updates. "
-                               "Using default value:" << _gitUserRepo;
+        qCWarning(NETWORKTOOL) << "Fail to get github user and repository! "
```

Comment:
should this one be qstringliteral?

Reply:
no, qDebug and friends don't need a QString and work witha  char *